x/sequencers/keeper/params.go

```diff
@@ -18,7 +18,6 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramstore.SetParamSet(ctx, &params)
 }
 
-// UnbondingTime
 func (k Keeper) UnbondingTime(ctx sdk.Context) (res time.Duration) {
```

Comment:
unused anymore?
same for `HistoricalEntries`

Reply:
isn't it out of scope?